src/core/Akka/Actor/FSM.cs

```diff
@@ -1325,6 +1325,7 @@ protected override void PostStop()
         /// <summary>
         /// By default, <see cref="Failure"/> is logged at error level and other
         /// reason types are not logged. It is possible to override this behavior.
+        /// Should handle the case when Cause is null and don't log. If that is not the desired behavior, LogTermination can always be overridden
```

Comment:
The summary tag is a documentation tag, it will be emmited into the API documentation. You should use a more affirmative language and tell the user what the method is actually doing.

Reply:
Removed that now and reverted back to the original